evap/staff/tests/test_views.py

```diff
@@ -2432,19 +2436,27 @@ def setUpTestData(cls):
 
     def test_emailtemplate(self):
         """
-            Tests the emailtemplate view with one valid and one invalid input datasets.
+            Tests the emailtemplate view with one valid and two invalid input datasets.
```

Comment:
I think its fair to just remove the comment, it doesn't add much value (however, now that you've changed it, I don't mind, we can also keep it)